src/bin/fullnode.rs

```diff
@@ -115,14 +115,19 @@ fn main() {
             if let Ok(data) = serde_json::from_reader(file) {
                 repl_data = data;
             } else {
-                warn!("failed to parse leader {}, generating new identity", path);
```

Comment:
I don't like thie behavior, would lobby for predictability.  i.e. if the file isn't there or is unparse-able, don't try to mask and do the "right thing", it only confuses a user who's mis-typed instructions to find an identity.

this is especially true in a project that is so verbose about info logging, warnings get lost in the noise, users who are making mistakes on the command line don't realize it, hell: can't realize it...

Reply:
yep, it bugged me a little too but decided that was out of scope for my first babystep Rust PR.  I filed #367 so we can follow up on this sometime